platform/authorized_api/user_groups/public_v1.yaml

```diff
@@ -235,7 +235,9 @@ paths:
     post:
       deprecated: true
       summary: Set permission group
-      description: Sets permission group to entity for user group
+      description: >
+        Please use <a href="?path=./access-control/public_v2.yaml#operation/set_app_permission_for_user_group_v2">Set user group permission for a given app</a> or <a href="?path=./access-control/public_v2.yaml#operation/set_meta_site_permission_for_user_group_v2">Set user group permission for a given meta site</a> instead.<br><br>
```

Comment:
This link is more likely suited to kopernik rather than developers.piwik.pro - please update it.

Reply:
maybe i will remove the links but leave endpoint names?

Reply:
Sounds fine to me